apache-jsp/src/test/java/org/eclipse/jetty/jsp/TestJettyJspServlet.java

```diff
@@ -56,23 +58,23 @@ public static class DfltServlet extends HttpServlet
         protected void doGet(HttpServletRequest req, HttpServletResponse resp) throws IOException
         {
             resp.setContentType("html/text");
-            resp.getOutputStream().println("This.Is.The.Default.");
+            resp.getOutputStream().println(DEFAULT_RESP_OUTPUT);
         }
     }
 
     @BeforeEach
     public void setUp() throws Exception
     {
         JspFactory.setDefaultFactory(new JspFactoryImpl());
-        File baseDir = MavenTestingUtils.getTestResourceDir("base");
+        final Path path = Path.of("src", "java", "resources", "base").toAbsolutePath();
```

Comment:
Do not change this to normal Path behaviors.
When we merge up, and the build/test gets parallelized the technique you are using will fail.
It has to use the `${project.base}` system property (when present) to find the directory correctly.

Use `Path baseDir = org.eclipse.jetty.toolchain.test.MavenPaths.findTestResourceDir("base");` instead if you want to get rid of the deprecation warning.